src/script/ceph-debug-docker.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# This can be run from e.g. the senta machines which have docker available. You
+# may need to run this script with sudo.
+#
+# Once you have booted into the image, you should be able to debug the core file:
+#     $ gdb -q /ceph/teuthology-archive/.../coredump/1500013578.8678.core
+#
+# You may want to install other packages (yum) as desired.
+#
+# Once you're finished, please delete old images in a timely fashion.
+
+set -e
+
+function run {
+    printf "%s\n" "$*"
+    "$@"
+}
+
+if [ -z "$1" ]; then
+    printf "specify the branch\n"
+    exit 1
+fi
+branch="$1"
+
+printf "branch: %s\n" "$branch"
+
+image="$(whoami):${branch}"
+
+time run docker build --tag "$image" - <<EOF
+FROM centos:7
+
+WORKDIR /root
+RUN yum update -y && \
+    yum install -y screen epel-release wget psmisc ca-certificates gdb
+RUN wget -O /etc/yum.repos.d/ceph-dev.repo https://shaman.ceph.com/api/repos/ceph/${branch}/latest/centos/7/repo && \
```

Comment:
the core dump is not always generated in a centos7 env. so maybe we need to enable the script to support xenial/trusty also?